packaging_automation/tests/files/citus.spec

```diff
@@ -7,12 +7,12 @@ Summary:	PostgreSQL-based distributed RDBMS
 Name:		%{sname}%{?pkginfix}_%{pgmajorversion}
 Provides:	%{sname}_%{pgmajorversion}
 Conflicts:	%{sname}_%{pgmajorversion}
-Version:	10.0.2.citus
+Version:	10.2.4.citus
```

Comment:
Yes copy paste error. Actually it should be 10.1.4 which is former version of 10.2.4. We are using this file to assure that 10.2.4 is being updated correctly. Since we are changing the line, having a wrong version in citus.spec file does not affect the tests. We simply change the version line